### ISSUE TITLE
EZPlaceholderHook deprecated

### ISSUE THREAD

**Issue by SlimeDog** [DenizenScript/Depenizen#258]
Spigot 1.13.2 final
PlaceholderAPI 2.10.0-b65
Depenizen 1.0.0-b-462

Per [PlaceholderAPI](http://ci.extendedclip.com/job/PlaceholderAPI/changes):
```
#65 (May 5, 2019 10:40:16 AM)
Remove EZPlaceholderHook which has been deprecated for over a year. (commit: 6495a38) — Ryan McCarthy / githubweb
``` 
The method has been replaced temporarily, but on server start-up
```
[08:14:14] [Server thread/WARN]: [PlaceholderAPI] Depenizen is currently using a deprecated method to hook into PlaceholderAPI. This will be disabled after the next update. Please consult [Morphan1, DenizenScript team] and urge them to update it ASAP.
```

**Comment by mcmonkey4eva** [comment]
Fixed for build 463+
Thanks for the report!

**Comment by SlimeDog** [comment]
Confirmed fixed on Spigot 1.13.2 final and Spigot 1.14 (2270) with Depenizen 1.0.0-b464.
